scripts/cerate_start_end_year_columns.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE courses
```

Comment:
Why isn’t these columns included in `create_courses.sql` file directly? Couldn’t you add these columns when the table is being created instead of alternating them after?